lib/meowing_cat.rb

```diff
@@ -1 +1,12 @@
 ## code your solution here. 
+class Cat
+  attr_accessor :name
+
+  def initialize(name)
+    @name = name
+  end
```

Comment:
There's no need for an initializer here since the `Cat` is never initialized with a name in the tests (besides where you updated it). 
```suggestion
```